clang/tools/pattern-rewriter/src/driver.cpp

```diff
@@ -16,11 +16,26 @@
 #include "llvm/Support/CommandLine.h"
 #include "llvm/Support/YAMLTraits.h"
 
-static constexpr auto HelpMessage = R"--(
-DESCRIPTION:
+static constexpr auto Description = R"--(
+
+SYCLomatic provides the pattern-rewriter tool to apply code updates and
+adjustments automatically based on user-defined patterns in YAML format. It can
+process any code before or after migration. For example, you can use it to
+automate the manual adjustments after migration to SYCL, enabling you to re-run
+migration multiple times and re-apply your adjustments.
+)--";
+
+static constexpr auto Examples = R"--(
+EXAMPLES:
+
+Rewrite SYCL source file:
+
+  pattern-rewriter source.dp.cpp -r rules.yaml -o output.dp.cpp
```

Comment:
please also give an content example for rules.yaml, maybe just several lines.   Here rules.yaml is unknown, while source.dp.cpp is generated from dpct,  and source.cu is from user project.